manifests/sp_configure.pp

```diff
@@ -53,4 +51,11 @@
     onlyif   => template('sqlserver/query/sp_configure.sql.erb'),
     require  => Sqlserver::Config[$instance]
   }
+
+  exec{"restart-service-${service_name}":
+    command     => template('sqlserver/restart_service.ps1.erb'),
+    provider    => powershell,
+    logoutput   => true,
```

Comment:
Is having `logoutput` on by default with no way to turn off a good idea? Doesn't that always add stuff to the logs in PE?

Reply:
I don't think I feel strongly that it's a good or a bad idea. It will add things to the logs. In testing I didn't notice it being especially verbose, but I wasn't exactly looking for it. I can test again to try and find the specific impact.

Reply:
Adding or removing doesn't seem to have an effect on the output, at least under normal operating circumstances.
```
Notice: Compiled catalog for jo3s142of0k3t3g.delivery.puppetlabs.net in environment production in 0.37 seconds
Notice: /Stage[main]/Main/Sqlserver_tsql[check advanced sp_configure]/returns: executed successfully
Notice: /Stage[main]/Main/Sqlserver::Sp_configure[MSSQLSERVER-max memory]/Sqlserver_tsql[sp_configure-MSSQLSERVER-max server memory (MB)]/returns: executed successfully
Notice: /Stage[main]/Main/Sqlserver::Sp_configure[MSSQLSERVER-max memory]/Exec[restart-service-MSSQLSERVER]: Triggered 'refresh' from 1 event
Notice: Applied catalog in 5.46 seconds
```